### DISCUSSION TITLE
No one can find my account even via profile URL

### DISCUSSION THREAD

**Discussion by aubrielee** [mastodon/mastodon#23047] [Q&A] [answered]
Hi all, last month, I started hosting an instance via DigitalOcean: verse.aubrielee.com. My main website, aubrielee.com, is hosted on GitHub Pages.

My profile is at https://verse.aubrielee.com/@Aubrie. However, friends on other servers cannot find my profile via search or any other means.

When I search for a friend and click Follow, the button converts to Withdraw follow request. But my friend does not see any follow requests on their side.

Why can't I be found by or follow anyone?

**Comment by rbairwell** [comment, marked as answer]
I think you have a WEB_DOMAIN or LOCAL_DOMAIN setting incorrect. (see https://docs.joinmastodon.org/admin/config/#federation )

Your profile gives the handle of @aubrie@aubrielee.com , but the aubrielee.com server does not provide a valid webfinger response ( https://webfinger.net/lookup/?resource=%40aubrie%40aubrielee.com ), even trying @aubrie@verse.aubrielee.com

**Reply by aubrielee** [reply]
In my .env production file, I set up:
LOCAL_DOMAIN=aubrielee.com
WEB_DOMAIN=verse.aubrielee.com
which seems to be what the mastodon documentation requires.

Thanks for noting the invalid webfinger response—didn't know of that lookup tool. I’m attempting to do the webfinger redirect via https://github.com/aubrielee/aubrielee.github.io/blob/master/.well-known/host-meta

I read on the [mastohost blog](https://masto.host/mastodon-usernames-different-from-the-domain-used-for-installation/) that static XML files might cause problems. Do you think this is the case here? If so, do you know of another way I could do the redirect via GitHub Pages?

**Reply by rbairwell** [reply]
For some reason, going to https://aubrielee.com/.well-known/host-meta gives a 404. - but that can be ignored as it isn't relevant.

As detailed at the bottom of the [WEB_DOMAIN ](https://docs.joinmastodon.org/admin/config/#federation)settings, you'll need to setup a webfinger redirect: however, Github pages being static means you are not able to do this. However, if this is a solo-user instance (i.e. just yourself), you can work around this by copying https://verse.aubrielee.com/.well-known/webfinger?resource=acct:aubrie@verse.aubrielee.com to https://github.com/aubrielee/aubrielee.github.io/blob/master/.well-known/webfinger (so any queries for any account on aubrielee.com will return your https://verse.aubrielee.com/@Aubrie one) which should work.

**Reply by aubrielee** [reply]
I believe that works—just tried following you (hope that's alright). Thank you so much for your help!
